Makefile

```diff
@@ -47,7 +47,7 @@ KVM_GO_VERSION ?= $(GO_VERSION:.0=)
 
 
 INSTALL_SIZE ?= $(shell du out/minikube-windows-amd64.exe | cut -f1)
-BUILDROOT_BRANCH ?= 2024.11.2
+BUILDROOT_BRANCH ?= 2025.02
```

Comment:
Why 2024.11.2 (what we used for other parts) is not right here?

Reply:
Well it is still about falco-modules(but it is not working, as you can see)

It seems that after we update the buildroot, buildroot will append the FALCO_MODULE_INSTALL_STAGING_OPTS and the FALCO_MODULE_INSTALL_TARGET_OPTS to cmake command, which causes a iso build failure. I don't know why this happens, so I tried some other versions. But it still cannot work. 

**I guess I will remove falco-modules for now.**

BTW I think we should still keep using 2025.02 because i found that in build root, 20xx.xx are version numbers for long term supports, while 20xx.xx.xx is not. If we want to update the buildroot, I think maybe LTS version is better

Reply:
LTS sounds good but we don't have to do this change now.

Reply:
uhh probably you misunderstood this change? The current buildroot version in main branch is `BUILDROOT_BRANCH ?= 2023.02.9`. 

Since we have to update this buildroot in this PR, why should't we update it to the latest LTS version?

Reply:
The pr message says "Update buildroot from 2023.02.9 to 2024.11.2"

Reply:
I guess I was also confused by the commit message: "fix falco-modules" - I assumed you  update falco version. I see now that this updates BUILDROOT_BRANCH.